zipline/testing/fixtures.py

```diff
@@ -481,10 +482,26 @@ class WithTradingEnvironment(WithAssetFinder,
     :class:`zipline.finance.trading.TradingEnvironment`
     """
     TRADING_ENV_FUTURE_CHAIN_PREDICATES = None
+    RESOURCE_DIR = join('tests/resources')
 
     @classmethod
     def make_load_function(cls):
-        return None
+        """
+        Load benchmark returns and treasury curves data from a static csv file.
+
+        This ensures that we have data over a consistent date range for the
+        entirety of the zipline tests, and guarantees that the data covers the
+        dates required for tests on both the equity and futures calendars.
+        """
+        benchmark_csv_path = join(cls.RESOURCE_DIR, '^GSPC_benchmark.csv')
```

Comment:
If you dislike that idea, then I'd at least like this to be robust to needing a wider range of data in the future. Right now tests would just fail again, like you were seeing.